scripts/check.py

```
@@ -189,7 +189,7 @@ def get_commit(files):
     if files == "commit":
         return "HEAD^"
 
-    if files == "main" or files == "master":
+    if files == "main" or files == "master" or files == "presto-0.293-clp-connector":
```

Pylint (3.3.7)

[refactor] Consider merging these comparisons with 'in' by using 'files in ('main', 'master', 'presto-0.293-clp-connector')'. Use a set instead if elements are hashable.

(R1714)